### PR TITLE
Completed CRUD, added home page

### PR DESCRIPTION
**Pull request by Gabrielle-Tobermann**
Link to Netlify: 
jurassic-world-dashboard.netlify.app

## Description
- We completed the rest of CRUD for all views. Now, users can create, update and delete cards.
- We are hiding buttons when the user is logged out. They can only read the information. 
- We created a modal for all of our forms (create and update).
- We fixed our duplicate bug (when user logged out and logged back in, cards were being created twice.) 
- We added a home page, with clickable cards which directs user to each view. 
- Added basic styling for home page cards. 
- Added a Jurassic World logo in the nav bar which redirects user to home page. 

## Related Issue
- #21
- #22
- #23 
- #24 
- #25 
- #42 
- #45
- #47 
- #55

## Motivation and Context
- CRUD is part of requirements for app functionality. 
- Hiding the buttons is necessary to prevent non users from creating, updating or deleting data. 
- Bug fix prevents cards from printing twice. 
- Home page enhances experience for user. 
## How Can This Be Tested?
- These can be tested in the Netlify version of our app. 
- Or by fetching our branch and running npm start. 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/javascripts/components/logoutButton.js [hunk 1]
```diff
@@ -1,8 +1,11 @@
 import firebase from 'firebase/app';
 import 'firebase/auth';
+import { domEventsListener } from '../helpers/events/domEvents';
 
 const signMeOut = () => {
+  document.querySelector('body').removeEventListener('click', domEventsListener);
   firebase.auth().signOut();
+  window.location.reload();
```
Comment (drteresavasquez) [inline src/javascripts/components/logoutButton.js]: Did you find this was necessary and if so, why?

File: src/javascripts/helpers/events/domEvents.js [hunk 2]
```diff
@@ -1,22 +1,312 @@
...
+import { emptyDinos, showDinos } from '../../components/cards/dinos';
+import addDinoForm from '../../components/forms/addDinoForm';
+import updateDinoForm from '../../components/forms/updateDinoForm';
+import { emptyEquipment, showEquipment } from '../../components/cards/equipment';
+import addEquipmentForm from '../../components/forms/addEquipmentFrom';
+import {
+  createEquipment, getSingleEquipment, updateEquipment, deleteEquipment, getEquipment
+} from '../data/equipmentData';
+import updateEquipmentForm from '../../components/forms/updateEquipmentForm';
+import formModal from '../../components/forms/formModal';
```
Comment (drteresavasquez) [inline src/javascripts/helpers/events/domEvents.js:33]: Whew! 33 lines of imports! Y'all getting fancy! LOL! 😄

**Review by drteresavasquez — APPROVED** [review]
Y'all are serious developers over there! Love seeing the commit history of who is doing what and that you are sticking to the milestones. This looks good. Just put the deploy link in the PR description and you are good to go!

File: src/javascripts/components/logoutButton.js (hunk 1 above)
Comment (Gabrielle-Tobermann) [reply src/javascripts/components/logoutButton.js]: Actually we tried that when we had issues with the duplicate cards but we can take it out. The removeEventListener is working. Would you like for us to delete it now and push the PR back up or can we delete in our next branch?

Comment (drteresavasquez) [reply src/javascripts/components/logoutButton.js]: It it is not needed, go ahead and remove it on this PR and once that is done, you can merge!

**Pull request merged**